src/openrct2-ui/windows/Ride.cpp

```diff
@@ -3651,6 +3665,42 @@ static void window_ride_operating_update(rct_window* w)
     }
 }
 
+static void window_ride_operating_set_min_waiting_time(rct_window* w, uint8_t time)
+{
+    auto rideSetSetting = RideSetSettingAction(w->number, RideSetSetting::MinWaitingTime, time);
+    GameActions::Execute(&rideSetSetting);
+}
+
+static void window_ride_operating_set_max_waiting_time(rct_window* w, uint8_t time)
+{
+    auto rideSetSetting = RideSetSettingAction(w->number, RideSetSetting::MaxWaitingTime, time);
+    GameActions::Execute(&rideSetSetting);
+}
+
+static void window_ride_operating_textinput(rct_window* w, rct_widgetindex widgetIndex, char* text)
+{
+    if ((widgetIndex != WIDX_MINIMUM_LENGTH && widgetIndex != WIDX_MAXIMUM_LENGTH) || text == nullptr)
+        return;
+
+    int32_t time32;
```

Comment:
We don't separate the declaration and definition anymore, unless necessary.

Reply:
Fixed.